apps/ledger-live-mobile/src/logic/postOnboarding/actions.ts

```diff
@@ -64,28 +67,20 @@ export const customImageAction: PostOnboardingAction = {
   ],
 };
 
-export const recoverAction: PostOnboardingAction = {
-  id: PostOnboardingActionId.recover,
-  Icon: Icons.ShieldCheck,
-  title: "postOnboarding.actions.recover.title",
-  titleCompleted: "postOnboarding.actions.recover.titleCompleted",
-  description: "postOnboarding.actions.recover.description",
-  buttonLabelForAnalyticsEvent: "Subscribe to Ledger Recover",
-  shouldCompleteOnStart: true,
-  getIsAlreadyCompleted: async ({ protectId }) => {
-    const recoverSubscriptionState = await getStoreValue("SUBSCRIPTION_STATE", protectId);
-
-    return recoverSubscriptionState === "BACKUP_DONE";
+export const syncAccountsAction: PostOnboardingAction = {
+  id: PostOnboardingActionId.syncAccounts,
+  featureFlagId: "llmLedgerSyncEntryPoints",
+  featureFlagParamId: "postOnboarding",
+  Icon: Icons.Refresh,
+  title: "postOnboarding.actions.syncAccounts.title",
+  titleCompleted: "postOnboarding.actions.syncAccounts.titleCompleted",
+  description: "postOnboarding.actions.syncAccounts.description",
+  actionCompletedPopupLabel: "postOnboarding.actions.syncAccounts.popupLabel",
+  buttonLabelForAnalyticsEvent: "Sync accounts",
+  getIsAlreadyCompleted: async ({ isLedgerSyncActive }) => {
+    return !!isLedgerSyncActive;
+  },
```

Comment:
I don't think getIsAlreadyCompleted serves much value? It at least doesn't need to be async anymore but can't isLedgerSyncActive just be used directly?